dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/FieldResource.java

```diff
@@ -295,12 +329,48 @@ public Response updateContentTypeFieldByVar(@PathParam("typeId") final String ty
 
 
 	@DELETE
+	@JSONP
+	@NoCache
+	@Produces({ MediaType.APPLICATION_JSON, "application/javascript" })
+	public Response deleteFields(@PathParam("typeId") final String typeId, final String[] fieldsID, @Context final HttpServletRequest req)
+			throws DotDataException, DotSecurityException {
+
+		final InitDataObject initData = this.webResource.init(null, false, req, false, null);
+		final User user = initData.getUser();
+		final FieldAPI fapi = APILocator.getContentTypeFieldAPI();
+
+		Response response = null;
+		try {
+			final List<String> deletedIds = new ArrayList<String>();
+
+			for (final String fieldId : fieldsID) {
+				try {
+					Field field = fapi.find(fieldId);
+					fapi.delete(field, user);
+					deletedIds.add(fieldId);
+				} catch (NotFoundInDbException e) {
+					continue;
+				}
+			}
+
+			final List<Field> contentTypeFields = fapi.byContentTypeId(typeId);
+			response = Response.ok(new ResponseEntityView(imap("deletedIds", deletedIds,
+					"fields", new JsonFieldTransformer(contentTypeFields).mapList()))).build();
+		}catch (Exception e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)